### ISSUE TITLE
a11y: add aria-required on required fields

### ISSUE THREAD

**Issue by giuliaghisini** [plone/volto#5619]

**Comment by Aarav238** [comment]
@giuliaghisini  Can you please help me with that issue , I thought issue is regarding `aria-required` and `aria-described` in normal html tags , but this issue is something else , so please help me toh get there.

**Comment by giuliaghisini** [comment]
> @giuliaghisini Can you please help me with that issue , I thought issue is regarding `aria-required` and `aria-described` in normal html tags , but this issue is something else , so please help me to get there.

When you are editing a content-type, for exaple, and you have some required fields in form, they doesn't have aria-required="true" attribute if it's a required field, and in case of some validation error, the error message isn't linked to the field, so the field has to have aria-described-by="id_of_the_wrapper_error_field_description" to enable screen reader to read error message for that field. 

This attributes are required for accessibility of the interface.

**Comment by Aarav238** [comment]
Thanks @giuliaghisini  I will work on it.

**Comment by praveen-rikhari** [comment]
I guess this is still not worked on? can i pick this one @giuliaghisini

**Comment by stevepiercy** [comment]
@praveen-rikhari please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do), [Contributing to Plone](https://6.docs.plone.org/contributing/index.html), and [Contributing to Volto](https://6.docs.plone.org/contributing/volto.html).

**Comment by giuliaghisini** [comment]
@Aarav238 have you done something about this? or is still an open issue?

**Comment by Aarav238** [comment]
I will fix ths. Its been a long time , want to contribute to this project again.

**Comment by Wagner3UB** [comment]
https://github.com/plone/volto/pull/7494
